index.html

```diff
@@ -120,179 +120,322 @@ <h3 class="project-title">Multi-Post stories</h3>
+      />
 
-        <div class="border">
-          <img src="images/Vector4.png" alt="line">
-        </div>
+      <div class="border">
+        <img src="images/Vector4.png" alt="line" />
+      </div>
 
-        <div class="language">
-          <img src="images/Rectangle55.png" alt="rectangle">
-          <h2 class="lang-header">Language</h2>
-          
-        </div>
-        <div class="skills">
-          <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>Css</li>
-          </ul>
-        </div>
-        <div class="language">
-          <img src="images/Rectangle56.png" alt="rectangle">
-          <h2 class="lang-header">Frameworks</h2>
-          
-        </div>
-        <div class="skills">
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-        </div>
-        <div class="language">
-          <img src="images/Rectangle57.png" alt="rectangle">
-          <h2 class="lang-header">Skills</h2>
-          
-        </div>
-        <div class="skills">
+      <div class="language">
+        <img src="images/Rectangle55.png" alt="rectangle" />
+        <h2 class="lang-header">Language</h2>
+      </div>
+      <div class="skills">
+        <ul>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </div>
+      <div class="language">
+        <img src="images/Rectangle56.png" alt="rectangle" />
+        <h2 class="lang-header">Frameworks</h2>
+      </div>
+      <div class="skills">
+        <ul>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
+      <div class="language">
+        <img src="images/Rectangle57.png" alt="rectangle" />
+        <h2 class="lang-header">Skills</h2>
+      </div>
+      <div class="skills">
+        <ul>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <section class="contact-section">
+      <div class="contact">
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+
+        <form action="https://formspree.io/f/mpzbdpaw" method="post">
           <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
+            <li class="contact-list">
+              <input
+                type="text"
+                name="fullname"
+                id="fullname"
+                placeholder="Full name"
+                maxlength="50"
+                required
+              />
```

Comment:
- [ ] Great job setting a `maxlength` attribute for this element, however, to be precise the requirement say it should be set to 30 please fix that.